excel-importer/src/test/java/uk/gov/hmcts/ccd/definition/store/excel/validation/SpreadsheetValidatorTest.java

```diff
@@ -244,7 +220,8 @@ public void shouldVaidate_WithAllWorkSheetsInPlace() {
         // This test case ould fail if an exception is thrown; nothing else to assert.
         validator.validate(definitionSheets);
 
-        // Could have used Latest AssertJ to assert no exception is thrown but it is not in POM
+        // Could have used Latest AssertJ to assert no exception is thrown but it is not
+        // in POM
```

Comment:
Looks like this could be moved back to the previous line as it doesn't appear to be exceeding the max number of characters on a line.